### PR TITLE
Fix the WE cross-os build

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

https://github.com/obscuronet/obscuro-internal/issues/1611
As per title.

### What changes were made as part of this PR

Removed the rpc -> evm dependency created by checking the EVM error type at the WE.

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: go/enclave/enclave.go [hunk 1]
```diff
@@ -11,44 +11,35 @@ import (
...
+	"github.com/obscuronet/go-obscuro/go/enclave/debugger"
 	"github.com/obscuronet/go-obscuro/go/enclave/events"
+	"github.com/obscuronet/go-obscuro/go/enclave/genesis"
+	"github.com/obscuronet/go-obscuro/go/enclave/l2chain"
 	"github.com/obscuronet/go-obscuro/go/enclave/mempool"
 	"github.com/obscuronet/go-obscuro/go/enclave/rollupmanager"
 	"github.com/obscuronet/go-obscuro/go/enclave/rpc"
+	"github.com/obscuronet/go-obscuro/go/ethadapter"
 	"github.com/obscuronet/go-obscuro/go/ethadapter/mgmtcontractlib"
+	"github.com/obscuronet/go-obscuro/go/responses"
```
Comment (otherview) [inline go/enclave/enclave.go:42]: Ordered the imports

File: go/rpc/encrypted_client.go [hunk 2]
```diff
@@ -7,25 +7,19 @@ import (
 	"fmt"
 	"reflect"
 
-	gethlog "github.com/ethereum/go-ethereum/log"
-
-	"github.com/ethereum/go-ethereum/rlp"
-
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/obscuronet/go-obscuro/go/common/log"
-	"github.com/obscuronet/go-obscuro/go/enclave/evm"
```
Comment (otherview) [inline go/rpc/encrypted_client.go:16]: Removed the EVM package and added the err util

File: go/rpc/encrypted_client.go [hunk 3]
```diff
@@ -7,25 +7,19 @@ import (
...
-
-	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/obscuronet/go-obscuro/go/common"
+	"github.com/obscuronet/go-obscuro/go/common/errutil"
+	"github.com/obscuronet/go-obscuro/go/common/log"
+	"github.com/obscuronet/go-obscuro/go/responses"
 
 	gethcommon "github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/crypto/ecies"
+	gethlog "github.com/ethereum/go-ethereum/log"
```
Comment (otherview) [inline go/rpc/encrypted_client.go:22]: Ordered the imports - no other change.

**Review by StefanIliev545 — APPROVED** [review]
looks about right

**Pull request merged**